python/lsst/pipe/analysis/fakesAnalysis.py

```diff
@@ -1173,4 +1264,102 @@ def fakesCompletenessPlot(inputFakes, inputFakesMatched, processedFakesMatched,
     description = "completenessHist"
     stats = None
     yield pipeBase.Struct(fig=fig, stats=stats, description=description, style="fakes")
-    plt.close()
+
+
+def fakesMagnitudePositionError(inputFakesMatched, processedFakesMatched, plotInfoDict, areaDict,
+                                raFakesCol="raJ2000_deg", decFakesCol="decJ2000_deg", raCatCol="coord_ra_deg",
+                                decCatCol="coord_dec_deg", magCol="base_PsfFlux_mag"):
+    """Makes plots showing the comparison between the input minus the extracted magnitudes and the
```

Comment:
Imperative mood please:  https://developer.lsst.io/python/numpydoc.html#writing-summaries-for-functions-and-methods